packages/core/src/Account.ts

```diff
@@ -434,7 +442,6 @@ export class Account<T = any> extends EventEmitter {
    */
   public async logout(clearData: boolean = false): Promise<void> {
     this.db?.close();
-    await this.secretsDb?.close();
```

Comment:
The secret DB can be closed as soon as the key has been generated/fetched for CoreCrypto, no need to wait that long with the DB open. 
It's now closed here https://github.com/wireapp/wire-web-packages/pull/4783/files#diff-0a7e6d4a8ee3b5e3153dbc2336fecd1741ea0f39e7226d6b7a5d63be31b2747fR330